dataprofiler/data_readers/data_utils.py

```diff
@@ -99,7 +114,11 @@ def unicode_to_str(data, ignore_dicts=False):
     return data
 
 
-def json_to_dataframe(json_lines, selected_columns=None, read_in_string=False):
+def json_to_dataframe(
+    json_lines: List[JSONType],
```

Comment:
We should actually validate this, @taylorfturner can `df = pd.DataFrame(json_lines)` take in any list of JSON?

Reply:
It works assuming `List[JSONType]` but should it be `Dict[str, List[JSONType]]`? 

```
from typing import Dict, List, Union
import pandas as pd

JSONType = Union[str, int, float, bool, None, List, Dict]

test_iter_list = []
test_iter_list.append(['test', 'test'])
test_iter_list.append([1,2,3])
test_iter_list.append([1.02, 2.02, 3.03])
test_iter_list.append([True, False])
test_iter_list.append([None, None])
test_iter_list.append([['test', 'test'], ['test', 'test']])
test_iter_list.append([{'test': 'test', 'test': 'test'}])


for test_iter in test_iter_list: 
    pd.DataFrame(test_iter)
```

Reply:
CC @JGSweets and @Sanketh7

Reply:
@Sanketh7 need to update the docstring for this on L126 `:type json_lines: list(dict)`

Reply:
> 
This does work. So I think we are GTG 
`for test_iter in test_iter_list: json_to_dataframe(test_iter)`

Reply:
List[list] works
```
In [307]: data = data_generator("""[["test", "test"], ["test", "test"]]""".splitlines())
In [308]: read_json(data)
Out[308]: [[['test', 'test'], ['test', 'test']]]
```